theano/printing.py

```diff
@@ -23,19 +23,33 @@
 from theano.compile.profilemode import ProfileMode
 
 pydot_imported = False
+pydot_imported_msg = ""
 try:
     # pydot-ng is a fork of pydot that is better maintained
     import pydot_ng as pd
     if pd.find_graphviz():
         pydot_imported = True
-except ImportError:
+    else:
+        pydot_imported_msg = "pydot-ng can't find graphviz. Install graphviz."
+except (ImportError, AttributeError):
```

Comment:
I don't think that's right, AttributeError should not be raised by pydot-ng, I mis-diagnosed that.